src/applications/disability-benefits/all-claims/pages/toxicExposure/toxicExposureChoicePage.jsx

```diff
@@ -0,0 +1,520 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import FormNavButtons from 'platform/forms-system/src/js/components/FormNavButtons';
```

Comment:
New custom page, but onChange it not adding conditions to pageData... Something is misaligned probably the data structure somewhere or just not plugged in all the way.